src/Tavisca.TMS.Persistence/Dal/TopicDal.cs

```
@@ -52,5 +52,68 @@ public bool Insert(TopicDto topic)
                 }
             }
         }
+        
+        public bool DeleteById(string id)
+        {
+            using (var conn = _connectionProvider.GetConnection())
+            {
+                try
+                {
+                    if (conn == null)
+                        return false;
+                    conn.Open();
+                    var query = "spDeleteTopicById";
+                    MySqlCommand command = new MySqlCommand(query, conn);
+                    command.CommandType = CommandType.StoredProcedure;
+                    command.Parameters.AddWithValue("@topic_id", id);
+                    var topicCount = command.ExecuteNonQuery();
+                    command.Dispose();
+                    if (topicCount == 0)
+                        return false;
+                    return true;    
+                }
+                catch (Exception e)
+                {
+                    return false;
```

why are we suppressing the exception